primo/utils/override_utils.py

```diff
@@ -19,42 +22,43 @@
 import pandas as pd
 
 # User defined lib
+from primo.data_parser.data_model import OptInputs
 from primo.utils.raise_exception import raise_exception
 
+LOGGER = logging.getLogger(__name__)
+
 
-class overridewidget:
+# pylint: disable = attribute-defined-outside-init
+class OverrideWidget:
```

Comment:
@ruonanli61 This is the standard naming convention (`PascalCase`) for classes

Reply:
Thank you for the comment!